jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -385,8 +386,15 @@ static JavaContainerBuilder getJavaContainerBuilderWithBaseImage(
       RawConfiguration rawConfiguration, ProjectProperties projectProperties)
       throws MainClassInferenceException, InvalidAppRootException, IOException,
           InvalidContainerizingModeException {
+    ContainerizingMode mode = getContainerizingModeChecked(rawConfiguration, projectProperties);
     AbsoluteUnixPath appRoot = getAppRootChecked(rawConfiguration, projectProperties);
 
+    if (mode == ContainerizingMode.NATIVE_IMAGE) {
```

Comment:
Can we output a warning here (EXPERIMENTAL), I don't want anyone to hold us to this implementation.

Reply:
Ok